client/src/containers/App/App.css

```diff
@@ -1,3 +1,24 @@
 .main-content {
   text-align: center;
 }
+.sidebar1 {
+  z-index: 1;
+  position: absolute;
+  max-width: 250px;
+  margin-top: 51px;
+}
+.page-wrapper1 {
+  min-height: 731px;
+  position: inherit;
+  margin: 0 0 0 250px;
+  padding: 0 30px;
+  border-left: 1px solid #e7e7e7;
+}
```

Comment:
Great job on styling these - did you try to see if they work ok on different screen sizes?

Reply:
It is  Haron's job.